codewof/templates/users/dashboard.html

```diff
@@ -30,11 +30,23 @@ <h2>Your questions for today</h2>
             <strong>Sorry!</strong> We currently don't have questions for you to do today.
           </div>
         {% endif %}
+        <p>You've solved {{num_questions_answered}} question{{ num_questions_answered|pluralize }} in the last month!</p>
```

Comment:
for consistency add a space before/after inside the curly braces (Also line 37, 45)